.eslintrc

```diff
@@ -1,7 +1,11 @@
 {
-  "extends": ["airbnb", "prettier", "prettier/react"],
+  "extends": ["airbnb", "prettier", "prettier/react", "plugin:jest/recommended"],
```

Comment:
This makes good sense, but be aware when updating monorepo that all packages relying on this will need to have this `eslint-plugin-jest` added now :)

Reply:
I also wanted to add `eslint-plugin-react-hooks` at some point, so could make sense to do it at the same time to minimise the work required?

Reply:
ah crap, indeed.
Any idea on how to fix this? Unless we split the eslintrc I don't see how we can achieve that.
Maybe it's ok that all packages have eslint-plugin-jest?

Reply:
I think it's fine, and hopefully we'll need it in the future anyway when we got time to add more tests!